### PR TITLE
[API docs] Fix docs.mender.io links

### PR DESCRIPTION
**Pull request by lluiscampos**

### CODE REVIEW

**Comment by lluiscampos** [comment]
Stopped the mender-qa Pipeline

File: docs/internal_api.yml [hunk 1]
```diff
@@ -228,7 +228,7 @@ paths:
       summary: Upload mender artifact
       description: |
         Upload mender artifact to a specific tenant. Multipart request with meta and artifact.
-        Supports artifact (versions v1, v2, and v3)[https://docs.mender.io/development/architecture/mender-artifacts#versions].
+        Supports artifact [versions v1, v2, 3](https://docs.mender.io/overview/artifact#versions).
```
Comment (alfrunes) [inline docs/internal_api.yml:231]: What happened to the `v` in `v3`?

File: docs/management_api.yml [hunk 2]
```diff
@@ -588,8 +588,7 @@ paths:
       summary: Upload mender artifact
       description: |
         Upload mender artifact. Multipart request with meta and artifact.
-
-        Supports artifact [versions v1, v2, 3](https://https://docs.mender.io/overview/artifact#versions).
+        Supports artifact [versions v1, v2, 3](https://docs.mender.io/overview/artifact#versions).
```
Comment (alfrunes) [inline docs/management_api.yml:591]: `v3`?

File: docs/internal_api.yml (hunk 1 above)
Comment (lluiscampos) [reply docs/internal_api.yml:231]: Ups. I just copied if from `management_api.yml`. I'll fix in both places.

**Comment by lluiscampos** [comment]
@alfrunes it should be ok now. Please check.

**Review by alfrunes — APPROVED** [review]

**Pull request merged**